src/erp/sales/SalesManager.java

```diff
@@ -22,4 +22,13 @@ public void generateInvoices() {
             System.out.println("Generated invoice for order: " + order.getOrderId());
         }
     }
+
+     // New method added for calculating monthly sales data
+    public List<String> getMonthlySalesData() {
+        List<String> salesData = new ArrayList<>();
+        salesData.add("Sale1: $1000");
+        salesData.add("Sale2: $1500");
+        salesData.add("Sale3: $2000");
```

Comment:
the idea of keeping sales amount as string is really beginners levels coding mistake. it will be really prone to mistake while converting